src/sql/src/plan/scope.rs

```diff
@@ -88,6 +88,11 @@ pub struct ScopeItem {
     /// variables in outer scopes that would otherwise be valid to reference,
     /// but accessing them needs to produce an error.
     pub lateral_error_if_referenced: bool,
+    /// If true, then this column represents an "exists" flag for the entire
+    /// row. In that case, this column must be excluded from `*` expansion. If the
```

Comment:
I think this could use some detail that it's used on the ordinality columns we contrive for table functions in scalar positions. Also "the entire row" was slightly confusing at first because it took me a second to realize it referred to the row generated by the table function and not the "entire row" returned to the client. idk if there's a clearer way of expressing it, but just flagging I got confused.